File: BackEnd-ProjectAPI/Sample/Sample.Infrastructure/EntityFramework/StockTransfer.cs
using System;
using System.Collections.Generic;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class StockTransfer
    {
        public StockTransfer()
        {
            StockTransferDetail = new HashSet<StockTransferDetail>();
        }

        public int Id { get; set; }
        public string TransferNo { get; set; } = null!;
        public DateTime TransferDate { get; set; }
        public int CompanyId { get; set; }
        public int FromWarehouseId { get; set; }
        public int ToWarehouseId { get; set; }
        public string? Remarks { get; set; }
        public string CreatedBy { get; set; } = null!;
        public DateTime CreatedDate { get; set; }
        public string? RecStatus { get; set; }

        public int BranchId { get; set; }
        public virtual Branch Branch { get; set; } = null!;
        public virtual ICollection<StockTransferDetail> StockTransferDetail { get; set; }
    }
}

File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/stockTransfer.ts
import type { StockTransferDetail } from "./stockTransferDetail";


export interface StockTransfer {
  id: number;
  transferNo: string;
  transferDate: Date;
  companyId: number;
  fromWarehouseId: number;
  toWarehouseId: number;
  remarks?: string | null;
  createdBy: string;
  createdDate: Date;
  recStatus?: string | null;
  stockTransferDetail?: StockTransferDetail[] | null;
}